### PR TITLE
additional  faqs

### PR DESCRIPTION
**Pull request by pablanopete**
adding some more faqs to troubleshooting

### CODE REVIEW

**Comment by chadyj** [comment]
Can you update the desktop FAQ link on /nightly to point to the desktop FAQ https://goo.gl/NB9n81

**Comment by pablanopete** [comment]
> Can you update the desktop FAQ link on /nightly to point to the desktop FAQ https://goo.gl/NB9n81

Why would we link out to a gdoc when we have a website with a "troubleshooting" section? Is this section no longer valid? https://status.im/build_status/desktop.html 

shouldn't we just update all of these FAQs to a website section? Something I'll be taking into consideration for my personal OKRs is a evolving FAQ document but I'd say we should move away from the google docs hyperlinks.

File: source/build_status/desktop_troubleshooting.md [hunk 1]
```diff
@@ -155,3 +155,29 @@ Time to time metro bundler for some reasons doesn't recognize changes in JavaScr
...
 On Linux metro bundler saves cache to the following directories (can be removed manually):
 /tmp/metro-cache-*
+
+### Where do I get Status Desktop?
+
+Nightlies are available from [here](https://status.im/nightly/)
+
+### How do I updgrade?  
+
+On a mac, Quit Status, download a new nightly, and drag to Applications
```
Comment (andytudhope) [inline source/build_status/desktop_troubleshooting.md]:
```suggestion
* On a Mac: quit Status, download a new nightly, and drag to the Applications Folder when prompted.
```

File: source/build_status/desktop_troubleshooting.md [hunk 2]
```diff
@@ -155,3 +155,29 @@ Time to time metro bundler for some reasons doesn't recognize changes in JavaScr
...
+On linux, just download new nightly and make sure it’s named “Status.AppImage”.
```
Comment (andytudhope) [inline source/build_status/desktop_troubleshooting.md]:
```suggestion
* On linux: just download new nightly and make sure it’s named “Status.AppImage”.
```

File: source/build_status/desktop_troubleshooting.md [hunk 3]
```diff
@@ -155,3 +155,29 @@ Time to time metro bundler for some reasons doesn't recognize changes in JavaScr
...
+
+### Status crashed and won’t Start. How to fix?
+
+On a mac, kill `ubuntu-server` in activity monitor and restart Status, restarting your machine should also make it boot if it's hanging.
```
Comment (andytudhope) [inline source/build_status/desktop_troubleshooting.md]:
```suggestion
On a Mac, open your Activity Monitor (Applications > Utilities > Activity Monitor), search for and kill any `ubuntu-server` process and then restart Status. Restarting your machine should also solve the issue.
```

File: source/build_status/desktop_troubleshooting.md [hunk 4]
```diff
@@ -155,3 +155,29 @@ Time to time metro bundler for some reasons doesn't recognize changes in JavaScr
...
+
+### I see a blank screen on Startup.  How do I fix it?
+
+On a mac, quit Status, then go to ~/Library/Application Support/ and delete any Status directories. Delete the app in ~/Application. Then reinstall.
```
Comment (andytudhope) [inline source/build_status/desktop_troubleshooting.md]:
```suggestion
* On a Mac: quit Status, then go to ~/Library/Application Support/ and delete any Status directories. Delete the app in ~/Applications. Then download from the nightlies page and reinstall.
```

File: source/build_status/desktop_troubleshooting.md [hunk 5]
```diff
@@ -155,3 +155,29 @@ Time to time metro bundler for some reasons doesn't recognize changes in JavaScr
...
+
+On linux, you should have gnomekeyring package installed to make keychain working. `libgnome-keyring0.so` should be available.  gnome-keyring requires additionally package `libgnome-keyring0` installed on linux system. As alternative currently we also support kWallet keychain, but gnome-keyring was tested more.
```
Comment (andytudhope) [inline source/build_status/desktop_troubleshooting.md]:
```suggestion
* On linux: you should have the `gnome-keyring` package installed to make your keychain work correctly. `libgnome-keyring0.so` should be available.  `gnome-keyring` also requires the package `libgnome-keyring0` to be installed on your system. As an alternative, we currently also support the `kWallet` keychain, but `gnome-keyring` has been tested more.
```

**Pull request merged**